pkg/codegen/convertutil/utilities.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convertutil
```

Comment:
I'm unsure of the desired behavior for an input like:
```
resource bucket1 "aws:s3:Bucket" {
	options {
 		version = "5.11.0"
 	}
 }
resource bucket2 "aws:s3:Bucket" {
	options {
 		version = "4.12.0"
 	}
 }
```

Can you clarify (preferably in comments on the  `convertutil` package) what the intended behavior is. Will the desired behavior change between languages?

Reply:
@AaronFriel I think we discussed lazily setting the highest package, even if there are multiple major versions, but I wonder if there is an alternative approach

Reply:
@iwahbe I think we don't support this _today_, & we ought to create an issue to support multiple versions of a package (by package name) in program generation & pulumi convert.